lib/jsdom/living/events/EventTarget.idl

```diff
@@ -1,6 +1,12 @@
 [Exposed=(Window,Worker)]
 interface EventTarget {
-  void addEventListener(DOMString type, EventListener? callback, optional boolean capture = false);
-  void removeEventListener(DOMString type, EventListener? callback, optional boolean capture = false);
+  void addEventListener(DOMString type, EventListener? callback, optional Options options);
```

Comment:
You need to ues the spec's IDL here: https://dom.spec.whatwg.org/#eventtarget

@Sebmaster, I don't suppose we support union types in webidl2js, do we?